pkg/deploy/api/v1/conversion.go

```diff
@@ -13,128 +14,6 @@ import (
-	out.ControllerTemplate.Replicas = in.Replicas
-	if in.Selector != nil {
-		out.ControllerTemplate.Selector = make(map[string]string)
-		for k, v := range in.Selector {
-			out.ControllerTemplate.Selector[k] = v
-		}
-	}
-	if in.Template != nil {
-		if err := s.Convert(&in.Template, &out.ControllerTemplate.Template, 0); err != nil {
-			return err
-		}
-	}
-	if err := s.Convert(&in.Strategy, &out.Strategy, 0); err != nil {
-		return err
-	}
-	return nil
-}
-
-func convert_v1_DeploymentStrategy_To_api_DeploymentStrategy(in *DeploymentStrategy, out *newer.DeploymentStrategy, s conversion.Scope) error {
-	if err := s.Convert(&in.Type, &out.Type, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.CustomParams, &out.CustomParams, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.RecreateParams, &out.RecreateParams, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.RollingParams, &out.RollingParams, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.Resources, &out.Resources, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.Labels, &out.Labels, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.Annotations, &out.Annotations, 0); err != nil {
-		return err
-	}
-	return nil
-}
-
-func convert_api_DeploymentStrategy_To_v1_DeploymentStrategy(in *newer.DeploymentStrategy, out *DeploymentStrategy, s conversion.Scope) error {
-	if err := s.Convert(&in.Type, &out.Type, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.CustomParams, &out.CustomParams, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.RecreateParams, &out.RecreateParams, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.RollingParams, &out.RollingParams, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.Resources, &out.Resources, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.Labels, &out.Labels, 0); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.Annotations, &out.Annotations, 0); err != nil {
-		return err
-	}
-	return nil
-}
-
-func convert_api_DeploymentTemplate_To_v1_DeploymentConfigSpec(in *newer.DeploymentTemplate, out *DeploymentConfigSpec, s conversion.Scope) error {
-	out.Replicas = in.ControllerTemplate.Replicas
-	if in.ControllerTemplate.Selector != nil {
-		out.Selector = make(map[string]string)
-		for k, v := range in.ControllerTemplate.Selector {
-			out.Selector[k] = v
-		}
-	}
-	if in.ControllerTemplate.Template != nil {
-		if err := s.Convert(&in.ControllerTemplate.Template, &out.Template, 0); err != nil {
-			return err
-		}
-	}
-	if err := s.Convert(&in.Strategy, &out.Strategy, 0); err != nil {
-		return err
-	}
-	return nil
-}
-
 func convert_v1_DeploymentTriggerImageChangeParams_To_api_DeploymentTriggerImageChangeParams(in *DeploymentTriggerImageChangeParams, out *newer.DeploymentTriggerImageChangeParams, s conversion.Scope) error {
 	out.Automatic = in.Automatic
```

Comment:
I guess it can. Since these are manual conversions I see why would we want this.